docs/troubleshooting/semgrep.md

```diff
@@ -24,17 +24,22 @@ We record Semgrep runtimes for each file and rule. This information is displayed
 
 ### I just want Semgrep to run faster
 
-Just run Semgrep with `--time` and not `--json`. This will output a list of the rules and files that took the longest. Oftentimes, users find that those files shouldn't have been scanned in the first place.
+Just run Semgrep with `--time` and not `--json`. This will output a list of the rules and files that took the longest. Often, you will find that those files shouldn't have been scanned in the first place.
 
-The first step to improving Semgrep's speed is limiting its run to only the files you care about. You can do this by adding a `.semgrepignore` file. See [how to ignore files & directories in Semgrep CI](/semgrep-ci/overview.md#ignoring-files-directories).
+The first step to improving Semgrep's speed is limiting its run to only the files you care about. You can do this by adding a `.semgrepignore` file. See [how to ignore files & directories in Semgrep CI](/ignoring-files-folders-code/).
 
-If you're still slow, you may want to examine the slowest rules. You may find that some of them don't apply to your codebase and can be skipped.
+If Semgrep is still running slowly, examine the output and identify the slowest rules. You may find that some of them don't apply to your codebase and can be skipped.
 
 ### I am a contributor who wants to improve Semgrep's engine
 
 #### Interpreting the result object
 
-For full timing information, run Semgrep with `--time` and `--json`. In addition, you will want to `time` the entire command to get the true wall time. Here is an example result object.
+For full timing information, run Semgrep with `--time` and `--json`. In addition, you will want to `time` the entire command to get the true wall time. The full command would look sometthing like:
+
+<pre class="language-bash"><code>time semgrep --config=auto <span className="placeholder">PATH/TO/SRC</span></code></pre>
+Substitute the optional placeholder <code><span className="placeholder">PATH/TO/SRC</span></code> with the path to your source code.
```

Comment:
One other suggestion here was to add the `-o output.json` to the full command, since it generates quite a long JSON file. Happy to add that if it makes sense.

Reply:
Yeah good idea It seems helpful to the user. Thank you!

Reply:
LOL, it's a good thing I came back to this since apparently I didn't add ANY of the arguments when I copy-pasted 😂 Fixing that now and will include the -o.